### PR TITLE
Form and mobile version final pull request

### PR DESCRIPTION
**Pull request by Habtamu210**
In this project:
- Forms are included
- Mobile version is finished

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is supposed to be worked on by you and your partner(s) but only your commit could be found on the commit history.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -332,61 +259,150 @@ img.placeholder {
...
+
+.framework {
+  padding-top: 15px;
+}
+
+/* contact us information */
+.contact {
+  background-color: #fff;
+  display: flex;
+  display: grid;
```
Comment (Olamarx) [inline style.css:285]: [OPTIONAL] You can remove the display flex as the display grid would be used since it is the latter.

**Review by Olamarx — APPROVED** [review]
Hi @Habtamu210, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Proper use of formSpree ✔️ 
- All linter passed ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Habtamu210) [reply style.css:285]: thank you I will

**Pull request merged**